pkg/ddevapp/ddevapp_test.go

```diff
@@ -3546,6 +3546,7 @@ func TestMultipleComposeFiles(t *testing.T) {
 	assert.NoError(err)
 	_, err = app.ReadConfig(true)
 	require.NoError(t, err)
+	app.DockerEnv()
```

Comment:
Added because the test didn't work if I only ran one test.

Caused by:

- #7012

Reply:
Actually, caused by #472 true? I think #7012 was unsuccessful?

Reply:
Yes.